paymentsheet/src/main/java/com/stripe/android/paymentsheet/BaseAddPaymentMethodFragment.kt

```diff
@@ -5,249 +5,302 @@ import android.view.LayoutInflater
-        viewBinding: FragmentPaymentsheetAddPaymentMethodBinding,
-        paymentMethods: List<SupportedPaymentMethod>,
-        initialSelectedItem: SupportedPaymentMethod
+    @FlowPreview
+    @Composable
+    internal fun PaymentElement(
+        enabled: Boolean,
+        supportedPaymentMethods: List<LpmRepository.SupportedPaymentMethod>,
+        selectedItem: LpmRepository.SupportedPaymentMethod,
+        showLinkInlineSignup: Boolean,
+        linkPaymentLauncher: LinkPaymentLauncher,
+        elementsFlow: Flow<List<FormElement>?>,
+        showCheckboxFlow: Flow<Boolean>,
+        onItemSelectedListener: (LpmRepository.SupportedPaymentMethod) -> Unit,
+        onLinkSignupStateChanged: (LinkPaymentLauncher.Configuration, InlineSignupViewState) -> Unit,
+        formArguments: FormFragmentArguments
     ) {
-        viewBinding.paymentMethodsRecycler.isVisible = true
-        viewBinding.paymentMethodsRecycler.apply {
-            setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
-            setContent {
-                val processing by sheetViewModel.processing
-                    .asFlow()
-                    .collectAsState(initial = false)
-                val selectedItem by sheetViewModel.getAddFragmentSelectedLpm()
-                    .asFlow()
-                    .collectAsState(initial = initialSelectedItem)
+        Column(modifier = Modifier.fillMaxWidth()) {
+            if (supportedPaymentMethods.size > 1) {
                 PaymentMethodsUI(
-                    selectedIndex = paymentMethods.indexOf(selectedItem),
-                    isEnabled = !processing,
-                    paymentMethods = paymentMethods,
-                    onItemSelectedListener = { selectedLpm ->
-                        if (sheetViewModel.addFragmentSelectedLPM != selectedLpm) {
-                            onPaymentMethodSelected(selectedLpm)
-                        }
-                    }
+                    selectedIndex = supportedPaymentMethods.indexOf(
+                        selectedItem
+                    ),
+                    isEnabled = enabled,
+                    paymentMethods = supportedPaymentMethods,
+                    onItemSelectedListener = onItemSelectedListener,
+                    modifier = Modifier.padding(top = 18.dp, bottom = 6.dp)
                 )
             }
-        }
-    }
 
-    @VisibleForTesting
-    internal fun onPaymentMethodSelected(paymentMethod: SupportedPaymentMethod) {
-        // hide the soft keyboard.
-        ViewCompat.getWindowInsetsController(requireView())
-            ?.hide(WindowInsetsCompat.Type.ime())
+            if (selectedItem.code == PaymentMethod.Type.USBankAccount.code) {
+                (activity as BaseSheetActivity<*>).formArgs = formArguments
+                Column(Modifier.padding(horizontal = 20.dp)) {
+                    AndroidViewBinding(FragmentAchBinding::inflate)
+                }
```

Comment:
Will we be able to just use a Composable here once https://github.com/stripe/stripe-android/pull/5589 is merged, or are there other things holding it up?